nuxeo-compound-documents-web/ftest/features/preview.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Check the preview of a Compound Document
+
+    I can see the preview of a Compound Document
+
+  Background:
+    Given user "John" exists in group "members"
+    And I login as "John"
+    And I have a Workspace document
+    And I have permission ReadWrite for this document
+    And I browse to the document
+    And I have a CompoundDocument imported from file "samplecompound.zip"
+    And I have permission Read for this document
+    And I browse to the document with path "/default-domain/my_document"
```

Comment:
Do we need to browse and then browse again in the next step?

Reply:
Ups, I think I made a wrong copy paste 😬 Let me review it.